src/api/users.js

```diff
@@ -94,7 +95,7 @@ export async function userPasswordResetRequest (ctx, email) {
   }
 
   try {
-    const user = await UserModelAPI.findOneAndUpdate({email}, updateUserTokenExpiry).exec()
+    const user = await UserModelAPI.findOneAndUpdate({ email }, updateUserTokenExpiry).exec()
```

Comment:
If we are making the login case insensitive perhaps the same should be done for the password reset